client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionEventForm/InteractionEventForm.tsx

```diff
@@ -53,16 +53,18 @@ const InteractionEventForm: React.FC<Props> = (
   });
 
   const initialInteractionDate = React.useRef<Date>(new Date(interactionData?.startTime as Date));
-
   const placeType = methods.watch(InteractionEventDialogFields.PLACE_TYPE);
   const placeSubType = methods.watch(InteractionEventDialogFields.PLACE_SUB_TYPE);
   const interactionStartTime = methods.watch(InteractionEventDialogFields.START_TIME);
-  const interationEndTime = methods.watch(InteractionEventDialogFields.END_TIME);
+  const interactionEndTime = methods.watch(InteractionEventDialogFields.END_TIME);
   const isUnknownTime = methods.watch(InteractionEventDialogFields.UNKNOWN_TIME);
   const locationAddress = methods.watch(InteractionEventDialogFields.LOCATION_ADDRESS);
   const placeName = methods.watch(InteractionEventDialogFields.PLACE_NAME);
   const placeDescription = methods.watch(InteractionEventDialogFields.PLACE_DESCRIPTION);
 
+  const [startTime, setStartTime] = useState<Date | null>(isNewInteraction ? null : interactionStartTime);
```

Comment:
Why do you need the state? all data is managed by react hook form

Reply:
If you want to know if the object has been changed you can use ```methods.formState.touched``` to decide that

Reply:
'all data is managed by react hook form ' -  This is the core of the bug: 
The requirement is to set an empty TimePicker (instead of 00:00),  the managed data - StartTime and EndTime are Date (not null) type. So I think the best way is to customize TimePicker's  state.

Reply:
wait..... if the state is initialized as null, how does the form know the data on the event 🤔